volttron/platform/web/admin_endpoints.py

```diff
@@ -41,7 +41,10 @@
 import re
 from urllib.parse import parse_qs
 
+import jwt
```

Comment:
why are we adding jwt to this module?

Reply:
No need. Leftover from a call to jwt.ExpiredSignatureError, which is now handled in the RemoteError.
Removed.